tests/system/actions.go

```diff
@@ -13,13 +14,24 @@ type akResult struct {
 	returnCode int
 }
 
+func splitToArgs(cmdArgs string) []string {
+	re := regexp.MustCompile(`\w+=".*?"|\w+=[\w.:]+|"[^"]+"|[\w-.:/]+`)
```

Comment:
What about supporting `'` as well?

Reply:
same note. It's not supposed to be a generic parser. The old one was splitting args by space, this one is trying to be just a bit more sophisticated since I need to parse a schedule expression

Reply:
OK